app/views/home/pricing.html.erb

```diff
@@ -1,2 +1,39 @@
-<h1>Home#pricing</h1>
-<p>Find me in app/views/home/pricing.html.erb</p>
+<section class="pricing-header bg-light px-3 py-3 pt-md-5 pb-md-4 mx-auto">
```

Comment:
Does not display well on mobile, see

![image](https://user-images.githubusercontent.com/8007722/71586732-326d6c80-2ae9-11ea-982d-644b983f851d.png)

Reply:
Fixed.